backend/pkg/transport/network/tcp/tcp_test.go

```diff
@@ -0,0 +1,86 @@
+package tcp_test
+
+import (
+	"encoding/json"
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/transport/network/tcp"
+	"github.com/rs/zerolog"
+	"os"
+	"testing"
+	"time"
+)
+
+type Socket struct {
+	SrcIP   string
+	SrcPort uint16
+	DstIP   string
+	DstPort uint16
+}
+
+// Payload defines a piece of information comming from the network with its metadata
+type Payload struct {
+	Timestamp time.Time
+	Socket    Socket
+	Data      []byte
+}
+
+func TestTCP(t *testing.T) {
+	writer := os.Stdout
+	logger := zerolog.New(writer)
```

Comment:
zerolog already provides a default logger that prints to stdout in the module `github.com/rs/zerolog/log`, although this is not too complex, use that one